datahub/bed_api/models.py

```diff
@@ -0,0 +1,249 @@
+import copy
+
+from datahub.bed_api.constants import (
+    BusinessArea,
+    Classification,
+    ContactType,
+    DepartmentEyes,
+    HighLevelSector,
+    InteractionType,
+    IssueType,
+    JobType,
+    LowLevelSector,
+    RegionOrInternational,
+    Salutation,
+    Sentiment,
+    TopIssuesByRank,
+    TransparencyStatus,
+)
+from datahub.bed_api.utils import remove_blank_from_dict
+
+
+class BedEntity:
+    """Represents a BED  base model"""
+
+    def __init__(self):
+        """Constructor for Id based BED entity"""
+        self.Id = None
+
+    def as_values_only_dict(self):
+        """
+        Generates dictionary with no empty values to optimise data being passed as a dictionary
+        Typical scenario is for new records being posted.
+        NOTE: If updating be sure to include all values, even blanks,
+                as blank may be valid on an update.
+        """
+        result = remove_blank_from_dict(self.as_all_values_dict())
+        return result
```

Comment:
```suggestion
        return remove_blank_from_dict(self.as_all_values_dict())
```
I don't think you really need the variable assignment here; in this context in python it's quite common just to omit the variable assignment in favor of an inlined `return`

Reply:
Good point - I did for debugging so I could see the value  but I will realign with just returning the value